CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ All notable changes to this project will be documented in this file.
 ## [2.1.5] - 2021-04-08
 
 ### Added
+- chore: Add custom user agent
```

Comment:
Typically we shouldn't include `chores` in CHANGELOGs to generate them we can use this tool: https://github.com/conventional-changelog/standard-version